pkg/internal/testabilities/testservices/fixture_arc.go

```diff
@@ -220,6 +221,19 @@ func (a *arcQueryFixture) WillReturnTransactionWithMerklePath(path sdk.MerklePat
 	return a
 }
 
+func (a *arcQueryFixture) WillReturnWithMindedTx() ARCQueryFixture {
+	tx := a.knownTransaction()
+	merklePath := mockValidMerklePath(a.TB, tx.txid)
+	blockHash, err := merklePath.ComputeRootHex(&tx.txid)
```

Comment:
It's not a block hash (use TestBlockHash like in the WillReturnTransactionWithMerklePath)